### PR TITLE
Edit help command

### PR DESCRIPTION
**Pull request by xinen26**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -155,29 +155,30 @@ Format: `/tag <ApplicationID> <Tag>`
...
 
 Examples:
-* `/comment 10 “S/Pass Holder”` will add the note “S/Pass Holder” to the applicant identified by ID 10.
+* `note 1 /note S/Pass Holder` will add the note “S/Pass Holder” to the applicant identified by ID 1.
 
-### Exporting interviewees contacts by pageId : `/export`
-Extracts interviewees contacts into a separate HRConnect identified by a pageId
+### Exporting applicants contacts by pageId : `/export`
+Extracts applicants' contacts into a separate HRConnect identified by a pageId
 Format: /export <ContactRange> <pageId>
```
Comment (ys112) [inline docs/UserGuide.md:182]: Note there is a change to how the current export works. Using only "export", it exports any contact shown in the current list.

For example, HR officer will first filter by role before sending "export" command which exports the applicants with filtered.

**Review by ys112 — COMMENTED** [review]
LGTM except the export command as there is a change to how it works.

File: docs/UserGuide.md (hunk 1 above)
Comment (xinen26) [reply docs/UserGuide.md:182]: Okay noted 👍 Thanks for pointing it out

File: docs/UserGuide.md [hunk 2]
```diff
@@ -177,14 +177,14 @@ Format: `note <ApplicationId> /note <Note>`
 Examples:
 * `note 1 /note S/Pass Holder` will add the note “S/Pass Holder” to the applicant identified by ID 1.
 
-### Exporting applicants contacts by role/stage : `/export`
-Filter to isolate the applicants that match the desired stage or role. Export will then extract those applicants' contacts into a separate HRConnect
+### Exporting applicants contacts : `/export`
```
Comment (lihaoquan) [inline docs/UserGuide.md:180]: The command is `export` without a '/' in front of it.

**Pull request merged**